Examples/Swift/UserLocationAnnotationExample.swift

```diff
@@ -112,9 +111,9 @@ class CustomUserLocationAnnotationView: MGLUserLocationAnnotationView {
     private func arrowPath() -> CGPath {
         let max: CGFloat = size / 2
         let pad: CGFloat = 3
-        
+
         let top =    CGPoint(x: max * 0.5, y: 0)
-        let left =   CGPoint(x: 0 + pad,   y: max - pad)
+        let left =   CGPoint(x: 0 + pad, y: max - pad)
```

Comment:
I’m not a fan of this type of change — the extra space was intentional, to align the second parameters vertically (to make the progression easier to read). Is there a way to avoid the linter in these situations?